src/loganalytics/azext_loganalytics/commands.py

```diff
@@ -0,0 +1,20 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+
+def load_command_table(self, _):
+
+    from azure.cli.core.commands import CliCommandType
+    from azext_loganalytics._client_factory import loganalytics_data_plane_client
+
+    loganalytics_sdk = CliCommandType(
+        operations_tmpl='azext_loganalytics.custom#{}',
+        client_factory=loganalytics_data_plane_client
+    )
+
+    with self.command_group('monitor loganalytics', loganalytics_sdk) as g:
+        g.command('query', 'execute_query')
```

Comment:
Use `g.custom_command()` to point directly at the custom namespace defined in your loader, no need to reference the same namespace again and will open up the possibility for specifying other sdks if needed.